docs/errors-and-limits.qmd

```diff
@@ -177,34 +177,48 @@ def intercode_ctf():
     )
 ```
 
-As with `message_limit`, it's also possible for a solver to set the `token_limit` directly on the `TaskState`:
+It's also possible for solvers, scorers or agents to limit their token usage, independently of the sample token limit:
```

Comment:
Let's put this in a section called `### Context Manager` and include this preface:

````markdown
::: callout-note
The `token_limit()` context manager described below is available
only in the development version of Inspect. To install the development
version from GitHub:

``` bash
pip install git+https://github.com/UKGovernmentBEIS/inspect_ai
```
:::
````